### ISSUE TITLE
Support for chunked HTTP?

### ISSUE THREAD

**Issue by polymorphic** [softwaremill/sttp#36]
Greetings --

I'm seeing the following error when working with an endpoint that responds with chunked HTTP:

HTTP chunk size exceeds the configured limit of 1048576 bytes

I use the AkkaHttpBackend backend w/ streaming so I assumed chunked http was handled out of the box. Did I miss something?

Thanks,

-Dragos

**Comment by adamw** [comment]
Yes, it *should* work out of the box. However, the default akka configuration still applies. Do you know if you indeed have chunks larger than 1048576 bytes?

Looking quickly here: http://doc.akka.io/docs/akka-http/current/java/http/configuration.html, I think setting `akka.http.parsing.max-chunk-size` to eg `10m` (the default is 1m, which would match the error message) in `application.conf`.

The config prefix might be different, `akka.http.client.parsing` or `akka.http.host-connection-pool.client.parsing`, I'm not sure if changes to the defaults are propagated

**Comment by adamw** [comment]
Great to hear that it works (for now ;) ) & thanks for checking out sttp!
